docker-compose.yml

```diff
@@ -1,48 +1,34 @@
 services:
+  pelacli:
+    container_name: pelacli
```

Comment:
Is cluster working fine? Transactions, their status?

Reply:
You mean cluster with example and pelacli ? Or the core cluster ?